doc/contributing.md

```diff
@@ -16,7 +16,7 @@ What can I do?
 You're in luck. There's various sources for tasks:
 
  - Have a look at the [issue tracker](https://github.com/sfttech/openage/issues), especially issues with labels:
-   - [good first issue](https://github.com/SFTtech/openage/labels/good first issue)!
+   - [good first issue](https://github.com/SFTtech/openage/labels/good%20first%20issue)!
```

Comment:
Could you exchange the link with this one, as this states explicitly that is has to be an open *issue* `https://github.com/SFTtech/openage/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22`

Reply:
It's done.